app/views/hyrax/dashboard/works/_list_works.html.erb

```diff
@@ -1,4 +1,8 @@
 <tr id="document_<%= document.id %>">
+  <td>
+    <label for="batch_document_<%= document.id %>" class="sr-only"><%=t("hyrax.dashboard.my.sr.batch_checkbox")%></label>
+    <%= render 'hyrax/batch_select/add_button', document: document %>&nbsp;
```

Comment:
What's the point of the `&nbsp` here?

Reply:
This was a copy and paste. It appears the `&nbsp;` is used to manipulate the vertical position of the checkbox inside the table cell (without it the checkbox renders below verticle center).